docs/yaml/elementary/dict.yml

```diff
@@ -11,6 +11,8 @@ description: |
   *(since 0.48.0)* Dictionaries can be added (e.g. `d1 = d2 + d3` and `d1 += d2`).
   Values from the second dictionary overrides values from the first.
 
+  *(since 0.62.0)* Dictionary order is guaranteed to be insertion order.
```

Comment:
Looks like you're missing a colon after the `since`

Reply:
done